src/style.css

```diff
@@ -1,3 +1,44 @@
-.hello {
-    color: blue;
-  }
\ No newline at end of file
+body {
+  padding: 5% 20%;
+}
+
+#todo {
+  background-color: white;
+}
+
+#todo-title,
+#add-div,
+#clear-div,
+li {
+  /* padding-bottom: 6px;
+ padding-top: 6px; */
+  height: 25px;
+  border-bottom: 1px solid rgb(238, 227, 227);
+}
+
+#todo-title {
+  font-weight: bold;
+}
+
+#add-div {
+  color: grey;
+  cursor: pointer;
```

Comment:
- [ ] Please work on the styles to match the provided design and use checkboxes for tasks instead of bullet points.

### Your design
![image](https://user-images.githubusercontent.com/83803414/206245241-e74d4a44-ba30-4f08-a19b-332d9948b42d.png)

### Provided design
![image](https://user-images.githubusercontent.com/83803414/206245286-709d350b-eb2f-4182-b00a-fbef5b69c9d7.png)